pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.coveo</groupId>
     <artifactId>spring-boot-parameter-store-integration</artifactId>
-    <version>1.3.1</version>
+    <version>1.4.0-SNAPSHOT</version>

```

Comment:
1.4.0 and don't forget the readme

Reply:
i can do the release if you want, if you do it, don't forget to do the release in github also

Reply:
It's just for snyk, I'll leave snapshot for now for some time